articles/azure-app-configuration/use-feature-flags-dotnet-core.md

```diff
@@ -103,7 +103,7 @@ The easiest way to connect your ASP.NET Core application to App Configuration is
               .UseStartup<Startup>();
    ```
 
-2. Open *Startup.cs* and update the `Configure` method to add the built-in middleware called `UseAzureAppConfiguration`. This middleware allows the feature flag values to be refreshed at a recurring interval while the ASP.NET Core web app continues to receive requests.
+2. Open *Startup.cs* and update the `Configure` and 'ConfigureServices' method to add the built-in middleware called `UseAzureAppConfiguration`. This middleware allows the feature flag values to be refreshed at a recurring interval while the ASP.NET Core web app continues to receive requests.
```

Comment:
To match the layout, could you please use the slanted apostrophes? Then it is  "and `ConfigureServices`"

Reply:
Yes, good point.

Reply:
> Yes, good point. Do you need me to re-issue the PR?

Hi @rpagels - You can edit this PR on the [diff view page](https://github.com/MicrosoftDocs/azure-docs/pull/69312/files):

![image](https://user-images.githubusercontent.com/19416435/105527354-71440680-5c98-11eb-85d6-3581978afbb9.png)

Then the author will review the PR again. Thanks.